### ISSUE TITLE
why need gt transoform in demo.py

### ISSUE THREAD

**Issue by MaybeOjbk** [qinzheng93/GeoTransformer#51]
in line 74 in experiments/geotransformer.3dmatch.stage4.gse.k3.max.oacl.stage2.sinkhorn/model.py , transform = data_dict['transform'].detach(), and  in line 110 you use gt transform in get_node_correspondences. however If i don't know how the  transform  matrix of src to ref, how can I use demo.py

**Comment by qinzheng93** [comment]
The ground-truth transform is mainly used to compute the registration error (RRE and RTE). And the ground-truth node correspondences are not used in the inference. If you do not need RRE and RTE, just give a `np.eye(4)`.

**Comment by MaybeOjbk** [comment]
yes, the ground-truth transform is used to compute the registration error, however, when I run demo.py and set gt transform to np.eye(4), the predicted transform is also np.eye(4), when  remaining the gt transform unchanged, the predicted transform have small changes

**Comment by MaybeOjbk** [comment]
first, I follow your instruction of demo.py, use gt transform 
![image](https://user-images.githubusercontent.com/38483419/204123133-1989d146-0ea8-4ede-bfd4-c7531edd85b5.png)
,the second  picture is that I changed the gt.npy, which is np.eye(4)
![image](https://user-images.githubusercontent.com/38483419/204123158-8e3d6718-2ab6-4954-aa08-b0d51eb36d3c.png)

**Comment by qinzheng93** [comment]
Sorry, I forget to set `model.eval()` in the demo script. Fixed in the latest commit.

**Comment by Parskatt** [comment]
@qinzheng93 Not sure if I'm missing something but the model seems to assume multiple things available in the data_dict that are not available still. For example, the coarse points are not extracted in the demo script (as far as I can tell).

**Comment by Parskatt** [comment]
It seems like I would need to apply a collate function on the data_dict in the demo before feeding it into the method, is this correct?
